enterprise/templates/enterprise/emails/user_notification.html

```diff
@@ -0,0 +1,10 @@
+<p>{% if user_name %}Dear {{ user_name }},{% else %}Hi!{% endif %}</p>
+<p>
+You have been enrolled in <a href="{{ course_url}}">{{ course_name }}</a>, a {% if program_type %}{{ program_type }} {% endif %}{{ course_type }} offered by {{ organization_name }}. This {{ course_type }} begins {{ start_date | date:"F j, Y" }}. For more information, see <a href="{{ course_url}}">{{ course_name }}</a>.
+</p>
+<p>
+Thanks,
+</p>
+<p>
+The {{course_name}} team
+</p>
```

Comment:
At the very least, the template should have `<html>`, `<body>` etc. Check out https://github.com/leemunroe/responsive-html-email-template for an example that's very nice and also where the code is easy to read, unlike the edx example.